sdk/communication/azure-communication-chat/src/test/java/com/azure/android/communication/chat/CommunicationSignalingClientTest.java

```diff
@@ -0,0 +1,51 @@
+package com.azure.android.communication.chat;
+
+import android.content.Context;
+
+import com.azure.android.communication.chat.implementation.notifications.signaling.CommunicationSignalingClient;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class CommunicationSignalingClientTest {
+
+    CommunicationSignalingClient communicationSignalingClient;
+
+    @Mock
+    Context context;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+    }
+
+    @Test
+    public void canParseTokenForRegion() {
```

Comment:
Is this test work-in-progress?

Reply:
Removed those files since parsing regions from the skype tokens are not needed from clients-side. Also removed other tests regarding parsing regions from tokens.